Content.Server/Objectives/ObjectivePrototype.cs

```diff
@@ -55,6 +79,13 @@ public void LoadFrom(YamlMappingNode mapping)
             ser.DataField(this, x => x._conditions, "conditions", new List<IObjectiveCondition>());
             ser.DataField(this, x => x._requirements, "requirements", new List<IObjectiveRequirement>());
             ser.DataField(this, x => x._difficultyOverride, "difficultyOverride", null);
+            ser.DataField(this, x=> x._incompatibleObjectives, "incompatible", new List<string>());
+            ser.DataField(this, x => x.CanBeDuplicateAssignment, "canBeDuplicate", false);
```

Comment:
For most of those you could just use `ref _privateVar`. No need for a expression if you don't have a property